<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpscaleDialog</class>
 <widget class="QDialog">
  <property name="windowTitle">
   <string>Upscale/Downscale</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox">
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Multiplier:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="LineEditWidget" name="multiplierLineEdit">
        <property name="charWidth">
         <number>3</number>
        </property>
        <property name="alignment">
         <enum>Qt::AlignCenter</enum>
        </property>
        <property name="toolTip">
         <string>'2', '3' ... to upscale, '1/2', '1/3' ... to downscale</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Fix colors:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="LineEditWidget" name="firstFixColorLineEdit">
        <property name="charWidth">
         <number>3</number>
        </property>
        <property name="alignment">
         <enum>Qt::AlignCenter</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel">
        <property name="minimumWidth">
         <number>8</number>
        </property>
        <property name="maximumWidth">
         <number>8</number>
        </property>
        <property name="text">
         <string notr="true">/</string>
        </property>
        <property name="alignment">
         <enum>Qt::AlignCenter</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="LineEditWidget" name="lastFixColorLineEdit">
        <property name="charWidth">
         <number>3</number>
        </property>
        <property name="alignment">
         <enum>Qt::AlignCenter</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QComboBox" name="levelTypeComboBox">
        <item>
         <property name="text">
          <string>Town</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>(L1) Cathedral</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>(L2) Catacombs</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>(L3) Caves</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>(L4) Hell</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>(L5) Crypt</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>(L6) Nest</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel">
        <property name="text">
         <string>Anti aliasing:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="4">
       <layout class="QHBoxLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="antiAliasingModeComboBox">
          <property name="sizeAdjustPolicy">
           <enum>QComboBox::AdjustToContents</enum>
          </property>
          <item>
           <property name="text">
            <string>Basic</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Tileset</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="upscaleButton">
       <property name="text">
        <string>Upscale</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="upscaleCancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4"/>
 <resources/>
 <connections/>
 <buttongroups/>
</ui>
